src/plugins/query_enhancements/public/query_assist/utils/errors.test.ts

```diff
@@ -42,6 +42,24 @@ describe('formatError', () => {
     expect(formattedError.message).toEqual(error.body.message);
   });
 
+  it('should return an AgentError if possible', () => {
+    const error = new ResponseError({
+      statusCode: 400,
+      body: {
+        statusCode: 400,
+        message: `{"error":{"reason":"Invalid Request","details":"PPLTool doesn't support searching indices starting with '.' since it could be system index, current searching index name: .opensearch-sap-log-types-config","type":"IllegalArgumentException"},"status":400}`,
```

Comment:
The last time I spoke with the documentation team, we want to use `indexes` for anything new we are adding. 

@opensearch-project/docs can u plz provide guidance.

Reply:
Yup, our style guide prescribes to use the word "indexes" (unless it's a variable; then "indices" is fine because we've inherited lots of settings/endpoints that use it in this form). So in error messages we should use "indexes".

Reply:
this is a unit test, the message came from the skills plugin. do we have any campaign to rename existing "indices" to "indexes" in all projects the org? interested to read more on the details

Reply:
I don't know if there is or was a campaign to replace existing ones but if anything new is being added, we wanna make sure to use `indexes`. The skills plugin sounds new; is it? If yes, we should let them know.

Reply:
based on [tags](https://github.com/opensearch-project/skills/tags) the oldest skills plugin is for OS_2.12

i think it would be good to have some documentation/guideline to align to